modules/administration-guide/pages/calculating-che-resource-requirements.adoc

```diff
@@ -13,38 +13,47 @@ The pods contribute to the resource consumption in CPU and memory limits and req
 Learn how to calculate resources, such as memory and CPU, required to run {prod}.
 
 .Procedure
-. Identify the workspace components explicitly specified in the `components` section of your devfile.
-When this section is empty, {prod-short} only loads the implicit components.
+
+. Identify the workspace resource requirements which depend on the devfile that is used for defining the development environment.
+
+As an example, we are going to use the following link:https://github.com/che-incubator/quarkus-api-example/blob/main/devfile.yaml[devfile] for calculating the resource requirements. Identify the workspace components explicitly specified in the `components` section of the devfile.
```

Comment:
According to our style guide, we shouldn't be using `we` in the docs:

![image](https://user-images.githubusercontent.com/78564304/227596056-4513a004-b836-48a5-a97f-5f933c73adc6.png)